CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [7.0.0](https://github.com/theforeman/puppet-pulpcore/tree/7.0.0) (2022-08-01)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/6.0.0...7.0.0)
+
+**Breaking changes:**
+
+- Drop EL7 support [\#257](https://github.com/theforeman/puppet-pulpcore/pull/257) ([ehelms](https://github.com/ehelms))
+
+**Implemented enhancements:**
+
+- Update to voxpupuli-test 5 [\#258](https://github.com/theforeman/puppet-pulpcore/pull/258) ([ekohl](https://github.com/ekohl))
+- Add Pulpcore 3.18 support [\#256](https://github.com/theforeman/puppet-pulpcore/pull/256) ([ianballou](https://github.com/ianballou))
```

Comment:
Is a minor release with this change present warranted at least? As this tends to try and match what Katello requires for its release

Reply:
I thought about that (https://github.com/theforeman/puppet-pulpcore/pull/257#pullrequestreview-1042295029) It is only a docs change, but I can do that just for consistency. We have shipped Katello 4.5 with Pulpcore 3.18 before without it though, so technically we're very late with it.